compiler/src/iree/compiler/Codegen/Common/TransformExtensions/CommonExtensionsOps.td

```diff
@@ -520,4 +520,38 @@ def TileToForallAndWorkgroupCountRegionOp :
   }];
 }
 
+def TileAndDecomposeAttentionOp : Op<Transform_Dialect, "iree.tile_and_decompose_attention",
+    [FunctionalStyleTransformOpTrait,
+     MemoryEffectsOpInterface,
+     TransformOpInterface,
+     TransformEachOpTrait]> {
+  let description = [{
+    Target iree_linalg_ext.attention ops and tile and decompose them.
+    This transform consumes the target handle and produces a result handle.
+  }];
+
+  let arguments = (
+      ins PDL_Operation:$target
+  );
+  let results = (outs Variadic<PDL_Operation>:$result);
+
+  let assemblyFormat = "attr-dict $target";
+  let cppNamespace = "mlir::iree_compiler::IREE::transform_dialect";
+
+  let builders = [
+    OpBuilder<(ins "Value":$target)>
+  ];
+
+  let assemblyFormat = [{
+    $target attr-dict `:` functional-type(operands, results)
+  }];
+
+  let extraClassDeclaration = [{
+    ::mlir::DiagnosedSilenceableFailure applyToOne(
+        ::mlir::iree_compiler::IREE::LinalgExt::AttentionOp target,
+        ::mlir::transform::ApplyToEachResultList &results,
+        ::mlir::transform::TransformState &state);
+  }];
+}
```

Comment:
should this live in `llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/TransformOps/LinalgExtTransformOps.td`?